### ISSUE TITLE
Crashes when explorer.exe is closed.

### ISSUE THREAD

**Issue by renns** [zgdump/windows-virtualdesktopindicator#14]
Repro:
* Go to task manager
* Kill explorer.exe
* See a bunch of error messages and then need to restart app

**Comment by vklachkov** [comment]
Hello. That's how it's intended. There used to be a restart, but it worked very poorly and was easier to remove than to fix

**Comment by Flaflo** [comment]
I removed the error message spamming in one of my PRs, maybe we should release the latest binary?

**Comment by renns** [comment]
Okay. Yeah, getting rid of the message spamming would be nice.

**Comment by vklachkov** [comment]
> I removed the error message spamming in one of my PRs, maybe we should release the latest binary?

Yes, I completely forgot to publish a new release with your changes. I'm sorry, I'll publish

**Comment by vklachkov** [comment]
Published a new release. But I leave the issue open so that I can return to it later
